Binary files RagTag-old/.git/index and RagTag/.git/index differ
diff -ruN RagTag-old/setup.py RagTag/setup.py
--- RagTag-old/setup.py	2021-06-23 15:39:09.445353653 +0200
+++ RagTag/setup.py	2021-06-23 15:38:48.621474180 +0200
@@ -8,7 +8,7 @@
 with open("README.md", "r") as fh:
     long_description = fh.read()
 
-scripts = glob.glob("*.p*")
+scripts = glob.glob("ragtag*.p*")
 
 version = get_ragtag_version()[1:]
 
@@ -37,4 +37,4 @@
     python_requires='>=3.6',
     scripts=scripts,
     zip_safe=True
-)
\ No newline at end of file
+)
